modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,61 @@
+Maintenance windows can be scheduled to suit your calendar.
+When you have created a schedule, you can select tasks that can only be performed during a scheduled maintenance window.
+
+Maintenance windows require both a calendar, and a schedule.
+The calendar defines the date and time of your maintenance window events, including recurring events, and must be in [path]``ical`` format.
+The schedule uses the events defined in the calendar to create the maintenance windows.
+You must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar, before you can create the schedule.
+
+When you have created the schedule, you can assign it to clients that are registered to the {productname} Server.
+Clients that have a maintenance window assigned cannot run restricted actions outside of the scheduled maintenance window.
+
+Restricted actions significantly modify the client, and could potentially cause the client to stop running.
+Restricted actions include:
+
+* Package installation
+* Client upgrade
+* Service pack migration
+* Highstate application (for Salt clients)
+
+Unrestricted actions are minor actions that are considered safe and are unlikely to cause problems on the client.
+Unrestricted actions include:
+
+* Package profile update
+* Hardware refresh
+* Subscribing to software channels
+
+
+Before you begin, you must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar.
+You can create [path]``ical`` files in your preferred calendaring tool, such as KOrganizer.
+// I don't know if this is correct. ---LKB 2020-07-21
+
+
+
+.Procedure: Creating a New Maintenance Calendar
+. In the {productname} {webui}, navigate to menu:Schedule[Maintenance Windows > Calendar], and click btn:[Create].
+. In the [guimenu]``Calendar Name`` section, type a name for your calendar.
+. Either provide a URL to your [path]``ical`` file, or upload the file directly.
+. Click btn:[Create Schedule] to save your calendar.
+// Well, that;s confusing :\ -- LKB 2020-07-21
+
+
+
+.Procedure: Creating a New Schedule
+. In the {productname} {webui}, navigate to menu:Schedule[Maintenance Windows > Calendar], and click btn:[Create].
+. In the [guimenu]``Schedule Name`` section, type a name for your schedule.
+. OPTIONAL: If your [path]``ical`` file contains more than one calendar, and you want to schedule all of the calendars, check [guimenu]``Multi``.
+. Select the calendar to assign to this schedule.
+. Click btn:[Update Schedule] to save your schedule.
+
+
+
+.Procedure: Assigning a Schedule to a Client
+. In the {productname} {webui}, navigate to menu:Systems[Systems List], select the client to be assigned to a schedule, locate the [guimenu]``Properties`` panel and click btn:[Edit These Properties].
+Alternatively, you can assign clients through the system set manager.
```

Comment:
Maybe I'd elaborate on the place of the feature in the SSM (it's a bit hidden there): it's in `<go to ssm> -> Misc -> Maintenance Windows`.